sopht_mpi/sopht_mpi_simulator/__init__.py

```diff
@@ -0,0 +1 @@
+from .flow import *
```

Comment:
^ditto

Reply:
Fixed in [3dc116b](https://github.com/fankiat/sopht-mpi/pull/53/commits/3dc116b6e70fd32f3b19f7077dd5d24da2a2d35e)

Reply:
Hmm, I'm not quite sure if I understand. I am following the implementation in https://github.com/SophT-Team/SophT-Examples/blob/main/sopht_simulator/__init__.py so that I can do `import sopht_mpi.sopht_mpi_simulator as sps` and call `flow_sim = sps.UnboundedFlowSimulator2D(...)`. Otherwise I think you have to do `from sopht_mpi.sopht_mpi_simulator.flow import UnboundedFlowSimulator2D` or something like that.

Reply:
Ah you are saying this relative to the `__init__.py` in `eulerian_grid_ops`. So this is fine here. Okay I think I got it. Thanks.